search-app/src/cmr/search/services/query_execution/facets/hierarchical_v2_facets.clj

```diff
@@ -11,14 +11,17 @@
    [cmr.common-app.services.search.parameters.converters.nested-field :as nested-field]
    [cmr.search.services.query-execution.facets.facets-results-feature :as frf]
    [cmr.search.services.query-execution.facets.facets-v2-helper :as v2h]
-   [cmr.search.services.query-execution.facets.hierarchical-links-helper :as hlh]))
+   [cmr.search.services.query-execution.facets.hierarchical-links-helper :as hlh]
+   [cmr.search.services.query-execution.facets.temporal-facets :as temporal-facets]))
 
 (defn- nested-fields-mappings
   "Returns nested field mappings for the given field, ignoring humanizer suffixes"
   [field]
   (let [stripped-field (string/replace (string/replace (name field) #"-h$" "") #"\.humanized$" "")]
-    (if (= "variables" stripped-field)
-      nested-field/variable-subfields
+    (condp = stripped-field
```

Comment:
Ive never thought about this kind of branching structure in clojure. Learn something everyday.

Reply:
Yeah, I love me some `condp`. Really keeps multi-conditionals clean.